runtime/j9vm/java11vmi.c

```diff
@@ -142,7 +142,10 @@ hashPackageTableDelete(J9VMThread * currentThread, J9ClassLoader * classLoader,
 	rc = hashTableRemove(table, &package);
 
 	if ((U_8 *) package.packageName != (U_8 *)buf) {
```

Comment:
fix formatting of near by code as per coding standard
```suggestion
	if ((U_8 *)package.packageName != (U_8 *)buf) {
```